### DISCUSSION TITLE
Missing packages for kirkstone release.

### DISCUSSION THREAD

**Discussion by bchoineubility** [bmwcarit/meta-ros#1165] [Q&A]
Hi Rob,

I updated my meta-ros by commit id 9a857afebc18e0f426d87226bd085727ea9ce4ae.  by the way, I found there are 4 packages are missed than before, commit id 950bcc85ad7a223252664688de71f9000cfeb61d.

Missed packages are :
```
aerostack2/as2-ign-gazebo-assets
aerostack2/as2-platform-ign-gazebo
dbw-ros/dataspeed-dbw-gateway
dbw-ros/dataspeed-dbw-msgs
```

Can you check and let me know whether it was intentional or not?

BR,
Mark

**Comment by robwoolley** [comment]
Those packages were removed from Humble by ROS.  The bitbake recipes were removed when the recipes were generated by this sync: https://github.com/ros/meta-ros/commit/efb3f96a57023a9ee1be7051141d42a5f0582cb1
